packages/vue2-jest/lib/utils.js

```diff
@@ -68,23 +70,55 @@ const getBabelOptions = function loadBabelOptions(filename, options = {}) {
   return loadPartialConfig(opts).options
 }
 
+const tsConfigCache = {}
+
 /**
  * Load TypeScript config from tsconfig.json.
  * @param {string | undefined} path tsconfig.json file path (default: root)
  * @returns {import('typescript').TranspileOptions | null} TypeScript compilerOptions or null
  */
 const getTypeScriptConfig = function getTypeScriptConfig(path) {
-  const tsconfig = loadTsConfigSync(process.cwd(), path || '')
-  if (!tsconfig.path) {
+  if (path in tsConfigCache) {
+    return tsConfigCache[path]
+  }
```

Comment:
Note: Return early from cache if we have already encountered that path.

I don't know how expensive `ts.getParsedCommandLineOfConfigFile` is, or if typescript is doing any caching internally, but I typically only call this once per-project rather than per source file when I've used it in the past. 

Since `getTypescriptConfig` is also called for each interpolated script block in the template as well as the `<script>` block of the SFC, it seemed like a good idea to just cache the parsed config result and reuse it.

Reply:
Nice